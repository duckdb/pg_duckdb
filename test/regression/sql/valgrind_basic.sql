CREATE EXTENSION pg_duckdb;

SELECT * FROM duckdb.query('SELECT 42 as answer');

SELECT * FROM duckdb.query('SELECT generate_series(1, 10) as num');

SELECT * FROM duckdb.query('SELECT ''Hello, Valgrind!'' as message');
