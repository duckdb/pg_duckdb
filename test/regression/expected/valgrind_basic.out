CREATE EXTENSION pg_duckdb;
SELECT * FROM duckdb.query('SELECT 42 as answer');
 answer
--------
     42
(1 row)

SELECT * FROM duckdb.query('SELECT generate_series(1, 10) as num');
 num
-----
   1
   2
   3
   4
   5
   6
   7
   8
   9
  10
(10 rows)

SELECT * FROM duckdb.query('SELECT ''Hello, Valgrind!'' as message');
     message
------------------
 Hello, Valgrind!
(1 row)
